docs/reference/query-dsl/query_filter_context.asciidoc

```diff
@@ -68,11 +68,16 @@ GET /_search
 <2> The `bool` and two `match` clauses are used in query context,
     which means that they are used to score how well each document
     matches.
-<3> The `filter` parameter indicates filter context.
-<4> The `term` and `range` clauses are used in filter context.
-    They will filter out documents which do not match, but they will
+<3> The `filter` parameter indicates filter context. Its `term` and 
+    `range` clauses are used in filter context. They will filter out 
+    documents which do not match, but they will
     not affect the score for matching documents.
 
+WARNING: Scores calculated for queries in query context are represented 
+as single precision floating point numbers; they have only
+24 bits for significand precision. Score calculations that exceed 
+this significand precision will be converted to floats with loss of precision. 
```

Comment:
same here.

Reply:
@cbuescher Thanks for the feedback, I have addressed your comments.